code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -367,23 +367,60 @@ SUBSYSTEM_DEF(dynamic)
-			advisory_string += "Your sector's advisory level is Yellow Star. Surveillance shows a credible risk of enemy attack against our assets in the Spinward Sector. We advise a heightened level of security alongside maintaining vigilance against potential threats."
-		if(66 to 79)
-			advisory_string += "Advisory Level: <b>Red Star</b></center><BR>"
-			advisory_string += "Your sector's advisory level is Red Star. The Department of Intelligence has decrypted Cybersun communications suggesting a high likelihood of attacks on Nanotrasen assets within the Spinward Sector. Stations in the region are advised to remain highly vigilant for signs of enemy activity and to be on high alert."
-		if(80 to 99)
-			advisory_string += "Advisory Level: <b>Black Orbit</b></center><BR>"
-			advisory_string += "Your sector's advisory level is Black Orbit. Your sector's local communications network is currently undergoing a blackout, and we are therefore unable to accurately judge enemy movements within the region. However, information passed to us by GDI suggests a high amount of enemy activity in the sector, indicative of an impending attack. Remain on high alert and vigilant against any other potential threats."
-		if(100)
-			advisory_string += "Advisory Level: <b>Midnight Sun</b></center><BR>"
-			advisory_string += "Your sector's advisory level is Midnight Sun. Credible information passed to us by GDI suggests that the Syndicate is preparing to mount a major concerted offensive on Nanotrasen assets in the Spinward Sector to cripple our foothold there. All stations should remain on high alert and prepared to defend themselves."
-
+	
+	//Part 1: Making the name of the level
+	var/advisory_name = ""
+	
+	//30% chance for a prefix
+	if (prob(30))
+		//The space is because one of them has a hyphen
+		advisory_name += pick("Bright ","Dark ","Light ","Old ","Natural ","Weird ","Alien ","Dim ","Pale ","Vivid ","Electric ","Powder ","Ruddy ","Neon ","Extremely ","Very ","Slightly ","Off-","Medium ","Russian ","French ","African ","Painfully ","Weirdly ","Rich ","Dusty ","Drab ","Marginally ")	
+	
+	//Initial color
+	advisory_name += pick("Burgundy","Carmine","Cinnabar","Crimson","Garnet","Mahogany","Maroon","Rust","Scarlet","Vermilion","Amaranth","Coral","Fuschia","Lilac","Rose","Peach","Salmon","Amber","Brown","Goldenrod","Saffron","Tangerine","Beige","Chartreuse","Citron","Cream","Ivory","Khaki","Marigold","Mustard","Aquamarine","Celadon","Emerald","Jade","Lime","Mint","Olive","Sage","Viridian","Azure","Cerulean","Teal","Turquoise","Blurple","Denim","Ice","Lapis","Lavender","Navy","Periwinkle","Sapphire","Ultramarine","Amethyst","Carnation","Cerise","Fuchsia","Grape","Indigo","Magenta","Mauve","Orchid","Plum","Violet","Auburn","Bone","Bronze","Caramel","Chocolate","Copper","Khaki","Sand","Tan","Charcoal","Ebony","Jet","Onyx","Vantablack","Silver","Slate","Gunmetal","Alabaster","Pearl","Platinum","Snow","Piss","Rainbow","Clear","Crystal","Transparent","Invisible")
```

Comment:
I feel like these should be .txt docs, would probably be cleaner

but then again april fools